irc/tests.py

```diff
@@ -6,11 +6,78 @@
 """
 
 from django.test import TestCase
+from .models import Irc, MSG_TYPES
 
+MSG_TYPES_STANDARD = (
+    ('M', 'Message'),
+    ('PM', 'Private Message'),
+    ('P', 'Part'),
+    ('J', 'Join'),
+    ('T', 'Topic'),
+    ('E', 'Error'),
+    ('N', 'Nick'),
+    ('Q', 'Quit'),
+    ('O', 'Other'),
+)
 
 class SimpleTest(TestCase):
     def test_basic_addition(self):
         """
         Tests that 1 + 1 always equals 2.
         """
         self.assertEqual(1 + 1, 2)
+
+class IrcViewTests(TestCase):
+    def test_login_loads_correctly(self):
+        """
+        Tests that the login page loads correctly
+        """
+        response = self.client.get('/irc/')
+        self.assertEqual(response.status_code, 200)
+
+    def test_login_reloads_correctly(self):
+        """
+        Tests that the login page reloads correctly when entering
+        the wrong password
+        """
+        response = self.client.post('/irc/', {'word': 'wrongpass'})
+        response = self.client.get('/irc/')
+        self.assertEqual(response.status_code, 200)
```

Comment:
Ta test se vedno nic ne pogleda. Fino bi blo, da bi pogledal v response.content, ce mas notr `<input type="password" name="word" placeholder="Enter the magic word.">`.